PrebidMobile/DemoApp/src/main/java/org/prebid/mobile/demoapp/dummyfragments/FBRequest.java

```diff
@@ -0,0 +1,214 @@
+package org.prebid.mobile.demoapp.dummyfragments;
+
+
+import android.content.Context;
+import android.os.AsyncTask;
+
+import com.facebook.ads.BidderTokenProvider;
+
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.io.OutputStreamWriter;
+import java.io.UnsupportedEncodingException;
+import java.net.HttpURLConnection;
+import java.net.MalformedURLException;
+import java.net.URL;
+
+public class FBRequest extends AsyncTask<Object, Object, JSONObject> {
```

Comment:
This file is just for testing right? Can it be removed yet?

Reply:
Will remove this after testing interstitial.